packages/db/drizzle/0014_lucky_captain_marvel.sql

```
@@ -0,0 +1,9 @@
+ALTER TABLE "update_requests" ALTER COLUMN "event_name" DROP NOT NULL;--> statement-breakpoint
+
+-- Convert any legacy 'create_location' values to 'edit' to preserve row compatibility
+UPDATE "public"."update_requests" SET "request_type" = 'edit' WHERE "request_type" = 'create_location';--> statement-breakpoint
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)